addons/website/static/src/js/website.js

```diff
@@ -304,14 +304,32 @@
                 if ($('html').data('website-id')) {
                     website.id = $('html').data('website-id');
                     website.session = new openerp.Session();
-                    var modules = ['website'];
-                    return openerp._t.database.load_translations(website.session, modules, website.get_context().lang);
+                    return openerp.jsonRpc( '/website/get_website_installed_module', 'call', {}).then(function (modules) {
```

Comment:
Create a controllers "/website/translations" who get the installed module and return the same result of "/web/webclient/translations" to have only one call.